com.jetbrains.RustRover.yml

```diff
@@ -0,0 +1,140 @@
+app-id: com.jetbrains.RustRover
+command: rustrover
+runtime: org.freedesktop.Sdk
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+separate-locales: false
+tags:
+  - proprietary
+
+finish-args:
+  - --allow=devel
+  - --device=all
+  - --env=RUST_ROVER_PROPERTIES=/app/bin/idea.properties
+  - --filesystem=host
+  - --filesystem=xdg-run/app/com.discordapp.Discord:create
```

Comment:
why does an IDE need access to discord?

Reply:
https://github.com/flathub/com.jetbrains.IntelliJ-IDEA-Community/pull/70

Reply:
Does this one have discord integration too? I'm not sure why you are linking to a separate app.

Reply:
As all Jetbrains IDE's are build based on the same foundations (with only having a focus on different languages/runtimes): yes. you can also install the discord extension in the Rust Rover IDE (see above ticket for IDEA community)